index.html

```diff
@@ -263,9 +231,10 @@ <h3 class='title'>Nearby Campaign Offices</h3>
             //Set filter info
             switch(d.event_type_name) {
               case "Phone Bank" : d.filters.push("phonebank"); break;
-              case "Canvass for Our Revolution" : d.filters.push("canvassing"); break;
               case "Volunteer Activity or Meeting" : d.filters.push("volunteer"); break;
               case "Party Meetings" : d.filters.push("party-meetings"); break;
+              case "Meet Senator Turner" : d.filters.push("meet-senator-turner"); break;
```

Comment:
I've only used switch/case syntax in Scala, didn't know you could do it in js too 🤔